website_cms/templates/page.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+  <template id="page_default_content" name="Page default content">
+    <t t-if="show_name">
+      <h1 id="object_name"
+         t-field="main_object.name"
+         t-att-contenteditable="editable and 'true' or None"
```

Comment:
looks like None is working too and I precisely used this to fix the issue with some browsers that allow you to move elements around even when not in edit mode.